docs/workflows/stories-for-multiple-components.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Stories for multiple components'
+---
+
+As discussed in an [earlier section](../writing-stories/introduction#stories-for-two-or-more-components), if you have sets of components that are designed to work together, it is convenient to write stories that render two or more components at once.
```

Comment:
```suggestion
It's useful to write stories that [render two or more components](../writing-stories/introduction#stories-for-two-or-more-components) at once if those components are designed to work together. For example, `ButtonGroups`, `Lists`, and `Page` components. 
```

Updated to not assume that the reader read the previous section.

Reply:
Marking this as resolved, applied the change